lib/hub/action_request.js

```diff
@@ -141,6 +141,15 @@ class ActionRequest {
                             winston.error(`[stream] request stream error`, Object.assign(Object.assign({}, this.logInfo), { error: err.message, stack: err.stack }));
                             reject(err);
                         }
+                    })
+                        .on("response", (response) => {
+                        if (response.statusCode !== 200) {
+                            winston.warn(`[stream] There was an error received from Looker.`, this.logInfo);
+                            winston.warn(`[stream] ErrorCode: ${response.statusCode} ErrorMessage: ${response.statusMessage}`);
```

Comment:
Should this be a single log message?

Reply:
It could but the log got so long it looked cleaner as two

Reply:
I think it's easier to use as one log line since it collects all the data in one place.